x-pack/osquerybeat/internal/distro/distro.go

```diff
@@ -41,14 +41,14 @@ const (
 
 	osqueryLensesDir = "lenses"
 
-	osqueryVersion = "5.10.2"
+	osqueryVersion = "5.12.1"
```

Comment:
any reason not to go with 5.12.2 which is the latest?

Reply:
<img width="677" alt="Screenshot 2024-07-26 at 3 50 31 PM" src="https://github.com/user-attachments/assets/51bca51f-36b6-4e2d-99d8-f444fabb3257">

Reply:
<img width="1202" alt="image" src="https://github.com/user-attachments/assets/713ade89-878f-401a-b478-ed007b26a2aa">
what am I misssing?!

Reply:
https://discuss.elastic.co/t/outdated-osquery-version-latest-5-12-1/363852

Reply:
ok sure, but still I see [here](https://github.com/osquery/osquery/releases/tag/5.12.2) that 5.12.2 is a hot fix release are you sure you don't wanna go with that from their github?!

Reply:
the discussion mentions 5.12.1, the official release on the osquery web is 5.12.1 https://www.osquery.io/downloads at the moment, 5.12.1 it is then.

Reply:
Another thing is that I don't see the checksums for the github releases and for some reason they didn't make it to the official releases page since May 8. 
This targets 8.16, we have time to pick up another update when it becomes official before 8.16 is out.